core/src/main/resources/application.yml

```diff
@@ -31,6 +31,8 @@ feast:
     # Key-value dict of job options to be passed to the population jobs.
     options: {}
     updates:
+      # Job update polling interval in milliseconds: how often Feast checks if new jobs should be sent to the runner.
+      pollingIntervalMillis: 60000
```

Comment:
Spring [has support for duration expressions](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration) to make these things more human-friendly, users could write `pollingInterval: 1m`.